### DISCUSSION TITLE
Introduction

### DISCUSSION THREAD

**Discussion by FishMacaron** [cooperrc/computational-mechanics#19] [General]
Hi, my name is Kaley and an interesting fact about me is that I've played violin since elementary school.

I am here to pursue my degree in Mechanical Engineering where I would like to work with the fun aspects of engineering such as animatronics. For my ideal career path, this will help by providing a way to easily calculate the physics of any project I'm working on though code as opposed to doing everything by hand.

If there was something I would change about this course it would be the option of video/auditory lessons per module since I personally do better with visually and auditorily taking in information when initially learning content.

**Comment by cooperrc** [comment]
> I would like to work with the fun aspects of engineering such as animatronics

so cool! Have you seen the [Disney Research team channel](https://www.youtube.com/user/DisneyResearchHub/videos)?
e.g. [Robot that walks like a cartoon character](https://youtu.be/6aqlosnp8nE)

> option of video/auditory lessons

I will keep posting short (10-20 min) videos based upon what questions come up and how we work through the assignments together. One thing that was massively helpful (in Spring) was to post recordings of live debugging sessions. I won't record our office hours unless you say its ok, but it really helps a lot of people if we can share the process. 
I hadn't considered audio lessons though. Could be a nice addition

**Reply by FishMacaron** [reply]
> so cool! Have you seen the [Disney Research team channel](https://www.youtube.com/user/DisneyResearchHub/videos)? e.g. [Robot that walks like a cartoon character](https://youtu.be/6aqlosnp8nE)

This is so cool! Most of Disney's work I've seen is their in-park animatronics like the [avatar](https://www.youtube.com/watch?v=_p4mn5BstQo) one and [Spiderman](https://www.youtube.com/watch?v=oyXl3IhonRM).

> I will keep posting short (10-20 min) videos based upon what questions come up and how we work through the assignments together. One thing that was massively helpful (in Spring) was to post recordings of live debugging sessions. I won't record our office hours unless you say its ok, but it really helps a lot of people if we can share the process.

Good to know, thank you!
